hackathon_site/event/jinja2/event/reset_password/reset_password_confirm.html

```diff
@@ -0,0 +1,33 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}{{ "Invalid Link" if form is none else "Reset Password" }}{% endblock %}
+
+{% block form_entry %}
```

Comment:
This was already possible without this block using the `form_action` and `form_method` variables, imo stick with that for consistency instead of having a second way to do it